NuGet.Config

```diff
@@ -8,6 +8,7 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/42249/Azure-Artifacts-on-the-azure-public-account?anchor=what-is-the-%60azure-public%60-account%3F

>  These feeds should not be used in a nuget.config file because particular package versions may be purged due to data retention policies.

You'll need to temporarily remove the `%40local` from the msft_consumption feed, do a restore with authentication, which will cause the packages to be copied into the consumption feed, then all the `%40local` back to the msft_consumption feed url.

Reply:
Had already discussed this with @nkolev92 and our understanding was that concern had been mitigated with archival feeds.

Reply:
Either way, thanks for the reminder, that process worked and it's now consumable from msft_consumption.